### ISSUE TITLE
Videos downloaded to the external sd card can't be played

### ISSUE THREAD

**Issue by designgrill** [TeamNewPipe/NewPipe#707]
If I download the videos to the external SD card on my phone through NewPipe, the download happens just fine but the App crashes when I try to play them back.

NewPipe 0.10
Android 7.0

Other apps are able to see and play those videos though.

**Comment by theScrabi** [comment]
But you can play those downloaded videos using other player right?

**Comment by designgrill** [comment]
@theScrabi apologies for a late reply, yes other players are able to play those videos.

**Comment by theScrabi** [comment]
I'm not sure if I understand right. NewPipe can not play downloaded videos by itself, you always have to open them with another app.

You mean if you select the video via NewPipe to play it in another app? Well if that causes an error its understandable, tho NewPipe should not even be able to download to an external sd card. I'm sorry we didn't add this feature yet.

**Comment by designgrill** [comment]
Ohk. I assumed it can play downloaded video all by itself because I first saw it stream the video in the Floating Player and assumed it has ability to play any video.

Coming to what's happening on my phone — NewPipe is able to download video the External SD Card without any issue if I set the download path accordingly in the settings. It even shows those videos in the "Downloads" screen though without thumbnail. When I try to "View" these videos, I get the App crash screen.

The videos downloaded are present on my SD card and I am able to then use some other video player to navigate to them and play.

Looks like some part of the feature (download videos to external SD card) already works, may be the team's magic touch can take it to completion soon enough 😄
